addons/grafana/dashboards/general-visualization.json

```diff
@@ -0,0 +1,1315 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "This dashboard allows you to check basically everything that can be seen in Porter dashboard, but if more details and control, being it:\n- App's CPU usage;\n- App's Memory usage;\n- App's number of instances;\n- App's amount of network data received;\n- App's number (and response status) of requests received;\n- App's logs\n- NGINX logs\n- Cluster's number of nodes",
```

Comment:
Maybe I would say `his dashboard allows you to check basically everything that can be seen in Porter dashboard, but if more details and control: App CPU usage, APP  Memory usage, etc.`